.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
Remove this please add it to your .git/info/exclude  file.

Reply:
You just taught me a new git trick - thanks!

And it is removed.